src/event.rs

```diff
@@ -55,5 +56,10 @@ pub enum Event {
     //    ~~|~~~~~~~~~~
     //      | our close group sorted from our name; always including our name
     //      | if size > 1, we are connected to the network
+    Connected,
+    Disconnected,
+    FailedRequest(Authority, ExternalRequest, InterfaceError),
+    FailedResponse(Authority, ExternalResponse, InterfaceError),
+
```

Comment:
Looks good to me, I think this is minimal info but required to allow client apps to know the current state. :+1: